idunn/api/places_list.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from elasticsearch import Elasticsearch
+from apistar.exceptions import BadRequest
+
+from idunn.utils.settings import Settings
+from idunn.utils.index_names import IndexNames
+from idunn.places import Place, Admin, Street, Address, POI
+from idunn.api.utils import get_geom, get_name, fetch_bbox_places, LONG, SHORT, DEFAULT_VERBOSITY_LIST, send_400
+
+from apistar import http
+
+logger = logging.getLogger(__name__)
+
+VERBOSITY_LEVELS = [LONG, SHORT]
+
+def bbox_or_400(verbosity, bbox):
```

Comment:
this function should be renamed.  
Or even better, why not validating each parameter separately ?